plugins/vintagestory/vintagestory.js

```diff
@@ -0,0 +1,651 @@
+(function () {
+    var codec;
+
+    var import_action;
+    var export_action;
+
+    var setting_gamepath;
+
+    var autosettings = [];
+    var namespace = {}
+
+    BBPlugin.register('vintagestory', {
+        title: "Vintage Story",
+        author: "Malik12tree,Crabb",
+        icon: "park",
+        description: "Export and import of Vintage Story shapes",
+        tags: ["Vintage Story"],
+        version: "1.0.2",
+        min_version: "4.9.2",
+        variant: "both",
+        await_loading: true,
+        creation_date: "2023-12-22",
+        onload() {
+            setting_gamepath = new Setting('vs_gamepath', {
+                name: 'Vintage Story game folder',
+                description: 'Set this to the install path',
+                category: 'defaults',
+                value: process.env.APPDATA.replaceAll('\\', '/') + '/Vintagestory',
+                type: 'text',
+            });
+
+            var format = new ModelFormat('vintagestory', {
+                id: "vintagestorymodel",
+                name: 'Vintage Story Model',
+                description: 'Model Format for VS specific features',
+                animated_textures: false,
+                animation_files: false,
+                animation_mode: true,
+                bone_binding_expression: false,
+                bone_rig: true,
+                box_uv: false,
+                category: 'general',
+                centered_grid: false,
+                display_mode: false,
+                edit_mode: true,
+                icon: 'park',
+                image_editor: false,
+                integer_size: false,
+                java_face_properties: true,
```

Comment:
These are enabled, but none of the properties are actually used (shade, cullface, tint).

Reply:
turned off, thought I needed these for the java texture window but appears not to be required